README.md

```diff
@@ -91,6 +91,7 @@ configuration:
             "default": [
                 "SecDebugLogLevel 9",
                 "SecRuleEngine On",
+                "Include @crs-setup-demo-conf",
```

Comment:
Giving a second thought I wonder if `@crs-setup-conf` is the right call here.

Reply:
Are you thinking about `@crs-setup-conf` (`crs-setup.conf.example`)?
`@crs-setup-demo-conf `could be more indicated if we are aiming to provide a slightly modified version more suitable for coraza-proxy-wasm (e.g. with early_blocking on). But I see that to keep it generic `@crs-setup-conf` could be more indicated.

First of all, I would make the [mapping](https://github.com/corazawaf/coraza-proxy-wasm/blob/8b1e1d2f24235530caeee7af72b466d8d925a368/wasmplugin/fs.go#L23) available directly from the readme